src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,308 @@
+    if (!triggerBinding) {
+      return TriggerBindingContainer.notification({
+        kind: 'error',
+        message: 'TriggerBinding not available',
+        intl
+      });
+    }
+
+    let labels = '';
+    const formattedLabelsToRender = [];
+    if (triggerBinding.metadata.labels) {
+      labels = JSON.stringify(triggerBinding.metadata.labels)
+        .replace('{', '')
+        .replace('}', '')
+        .replace(/['"]+/g, '')
+        .replace('$', '');
+
+      const labelsSplitOnComma = labels.split(',');
+      labelsSplitOnComma.forEach(foo => {
+        const splitOnColon = foo.split(':');
+        formattedLabelsToRender.push(`  ${splitOnColon[0]}: ${splitOnColon[1]}
+ `);
+      });
+    }
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'value',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerBinding.value',
+          defaultMessage: 'Value'
+        })
+      }
+    ];
+
+    const rowsForParameters = triggerBinding.spec.params.map(
+      ({ name, value }) => ({
+        id: name,
+        name,
+        value
+      })
+    );
+
+    const emptyTextMessage = intl.formatMessage({
+      id: 'dashboard.triggerBinding.noParams',
+      defaultMessage: 'No parameters found for this TriggerBinding.'
+    });
+
+    return (
+      <div className="triggerBinding">
+        <h1>{triggerBindingName}</h1>
+        <Tabs
+          onSelectionChange={index => {
+            this.handleTabSelect(index);
+          }}
+        >
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.triggerBinding.details',
+              defaultMessage: 'Details'
+            })}
+          />
+          <Tab label="YAML" />
+        </Tabs>
+        {showDetails && (
+          <div className="details">
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.dateCreated',
+                  defaultMessage: 'Date Created:'
+                })}
+              </span>
+              <FormattedDate
+                date={triggerBinding.metadata.creationTimestamp}
+                relative
+              />
+            </p>
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.labels',
+                  defaultMessage: 'Labels:'
+                })}
+              </span>
+              {formattedLabelsToRender.length === 0
+                ? intl.formatMessage({
+                    id: 'dashboard.triggerBinding.noLabels',
+                    defaultMessage: 'None'
+                  })
+                : formattedLabelsToRender.map(label => (
+                    <Tag
+                      type="blue"
+                      key={`${triggerBindingName}:label:${label}`}
```

Comment:
`label` would be fine as long as it's unique
the `key` only needs to be unique for this specific use of `map`, not globally